app/Nova/Actions/ActivateIntegration.php

```diff
@@ -36,13 +35,10 @@ public function handle(ActionFields $fields, Collection $integrations): ActionRe
         $organizationIdAsString = $fields->get('organization');
         $organizationId = Uuid::fromString($organizationIdAsString);
 
-        /** @var string $couponCode */
-        $couponCode = $fields->get('coupon');
-
         $this->integrationRepository->activateWithOrganization(
             Uuid::fromString($integration->id),
             $organizationId,
-            $couponCode
+            null
```

Comment:
Do we have to do this? I would not change this code. If the field is empty (because it is not included in the form, it should default to null anyway.